### DISCUSSION TITLE
Dynamic Value Discovery with Configured Filters

### DISCUSSION THREAD

**Discussion by Aminehsppro** [microsoft-search/pnp-modern-search#3333] [Q&A]
**When adding a configured filter on the "refinablerstring XX," it is possible that some values or all the values in the list may not be immediately found. However, as the search encounters a file, the values from that file start to gradually appear.**

**Comment by kasperbolarsen** [comment]
Really hard to assist when not following the issue template and no description nor screen shots. Please provide more info.
